pom.xml

```diff
@@ -13,7 +13,7 @@
   <properties>
     <revision>1020.140.0</revision>
     <changelist>-SNAPSHOT</changelist>
-    <c8y.version>${project.version}</c8y.version>
+    <c8y.version>1020.10000.0-SNAPSHOT</c8y.version>
```

Comment:
Probably unintended...

Reply:
of course :), but this PR is unfortunately not suitable for review yet.